spec/integration/inventory2_spec.rb

```diff
@@ -190,6 +190,30 @@ def fact_plan(name = 'facts_test')
     end
   end
 
+  context 'when creating targets from a plan with empty inventory', ssh: true do
```

Comment:
This seems like it could also test `set_config`. I think it'd also be useful to add a test that creating a node and adding it to a group picks up all the right data